1-exercises/B-setting-values/exercise1.js

```diff
@@ -24,6 +24,12 @@ let capitalCities = {
 
 // write code here
 
+capitalCities.China.population = 9750000;
```

Comment:
Gist of this is right, not quite what the question asked for but you obviously know what you're doing!